elastic/integrations
# Update Kibana integration to collect task manager HTTP metrics

## What does this PR do?


Updates the Kibana HTTP integration to collect task manager SLI metrics from the Kibana API endpoint added in [this PR](https://github.com/elastic/kibana/pull/163652).

## Checklist


## Author's Checklist



## How to test this PR locally



- Run the stack using `elastic-package`: `elastic-package stack up --version=8.10.0-SNAPSHOT -v -d`
- Build the package:
  - `cd integrations/packages/kibana`
  - `elastic-package build`
- Deploy the package: `elastic-package stack up -v -d --services package-registry`
- Navigate to https://localhost:5601/app/integrations/browse
- Select and add the Kibana integration
- Verify you see the new `HTTP metrics > Kibana task manager metrics` integration
- Configure the integration for host `https://kibana:5601` and username/password `elastic:changeme`
- Verify you see a datastream for `metrics-kibana.task_manager_metrics-default`
- Verify you see data in the datastream (`GET metrics-kibana.task_manager_metrics-default/_search`) with the `kibana.task_manager_metrics.metrics` field populated

## Related issues


-

## Screenshots
